plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThreadGroup.java

```diff
@@ -435,6 +435,16 @@ private boolean run() {
                     if (fn != null) {
                         t.head.get().addAction(new ErrorAction(error));
                     }
+                    if (error instanceof VerifyError) {
+                        var msg = error.getMessage();
+                        if (msg != null && msg.contains("Illegal type in constant pool")) {
+                            try {
+                                execution.getOwner().getListener().getLogger().println("May be JENKINS-73031: static interface methods not yet supported in Groovy");
```

Comment:
I would maybe rephrase this to indicate that it is more of a Jenkins problem than a Groovy problem:

```suggestion
                                execution.getOwner().getListener().getLogger().println("May be JENKINS-73031: static interface methods are not currently supported");
```

Reply:
Good point. Or what about
```suggestion
                                execution.getOwner().getListener().getLogger().println("May be JENKINS-73031: calling static interface methods from Groovy is not currently supported by Jenkins");
```
?

Reply:
Yes that looks good to me.